configs/gtp-amcts.cfg

```diff
@@ -17,10 +17,14 @@ useGraphSearch1 = true
 # This is set to 6 in the original gtp_example.cfg but A-MCTS only supports numSearchThreads = 1
 numSearchThreads = 1
 
+# conservativePass is "a good idea when using [the bot] for analysis or playing
+# on servers where scoring may be implemented non-tromp-taylorly", but we don't
+# care about this and are only trying to win under Tromp-Taylor rules.
+conservativePass = false
 # If we want to keep behavior close to KataGo v1.10, consider setting the
 # following parameters:
-# valueWeightExponent0 = 0.5
-# subtreeValueBiasFactor0 = 0.35
-# subtreeValueBiasWeightExponent0 = 0.8
-# fpuParentWeightByVisitedPolicy0 = false
-# enablePassingHacks0 = false
+# valueWeightExponent = 0.5
+# subtreeValueBiasFactor = 0.35
+# subtreeValueBiasWeightExponent = 0.8
+# fpuParentWeightByVisitedPolicy = false
+# enablePassingHacks = false
```

Comment:
fixing these params slightly — want to tweak these params not just for the adv but also for the adv's model of the victim. this actually does affect behavior, I think was seeing worse GTP win rates otherwise